cardano-cli/src/Cardano/CLI/Shelley/Run/Query.hs

```diff
@@ -846,10 +851,12 @@ determineEra cModeParams localNodeConnInfo =
     ByronMode -> return $ AnyCardanoEra ByronEra
     ShelleyMode -> return $ AnyCardanoEra ShelleyEra
     CardanoMode -> do
-      eraQ <- liftIO . queryNodeLocalState localNodeConnInfo Nothing
-                     $ QueryCurrentEra CardanoModeIsMultiEra
```

Comment:
We no longer need `queryNodeLocalState` because they can all by replaced by `executeLocalStateQueryExpr`/`queryExpr`.